mw-webapp/src/logic/wayPage/wayStatistics/statisticLabels/statisticLine/StatisticLine.module.scss

```diff
@@ -1,36 +1,43 @@
 @import "src/styles/variables";
 
+$labelWidth: 75px;
+$percentageBLockWidth: 40px;
+$labelColorSize: 10px;
+
 .statisticLine {
-  border-bottom: 1px dotted var(--secondaryStrokeColor);
-  gap: $gapSmall;
+  align-items: center;
+  border-bottom: $primaryBorder dotted var(--secondaryStrokeColor);
```

Comment:
We have component Separator? lets try to use it instead of border?

Reply:
done